### ISSUE TITLE
Cant get TXT Record of some Domains

### ISSUE THREAD

**Issue by 2peter3** [phuslu/fastdns#4]
Hey,
in short term, im trying to use fastdig as an alternative for "dig". When im using "dig" everything works expected, but on fastdig the result is missing. Example:

```
; <<>> DiG 9.16.44-Debian <<>> @1.1.1.1 t3n.de TXT
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 21655
;; flags: qr rd ra ad; QUERY: 1, ANSWER: 6, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;t3n.de.				IN	TXT

;; ANSWER SECTION:
t3n.de.			20995	IN	TXT	"nJg0mOQ"
t3n.de.			20995	IN	TXT	"v3nk3wg16gqgph0x4q2g45kj1j6g0b9f"
t3n.de.			20995	IN	TXT	"loaderio=6cead61edf5d0ea1a2297c1e7a409e87"
t3n.de.			20995	IN	TXT	"google-site-verification=ojkBs6li65pAGMwyGZl3Q1O0Q4DDMn8CPebe5bgeGNo"
t3n.de.			20995	IN	TXT	"hns-claim:aakgrn5php6ppybp6bjmlczduk3gplljzyfp5kfgaeaaaaaaaaaklzaororjdpl6qzexi75h7ofhv447lownw5btzuxvs4ibaaaabnvuzysq"
t3n.de.			20995	IN	TXT	"v=spf1 include:servers.mcsv.net include:spf.mailjet.com mx include:helpscoutemail.com include:_spf.intan.net include:_spf.google.com include:eu.zcsend.net ~all"

;; Query time: 4 msec
;; SERVER: 1.1.1.1#53(1.1.1.1)
;; WHEN: Wed Mar 20 15:40:46 EDT 2024
;; MSG SIZE  rcvd: 538

```

```
; <<>> DiG 0.0.1-fastdns-go1.21.5 <<>> t3n.de
;; global options: +cmd +noedns
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 65246
;; flags: qr tc rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;t3n.de.		IN	TXT

;; AUTHORITY SECTION:

;; Query time: 7 msec
;; SERVER: 1.1.1.1#53(1.1.1.1)
;; WHEN: Wed Mar 20 15:40:36 EDT 2024
;; MSG SIZE  rcvd: 24

```

Im using the master, maybe its a bug?

//edit: 
i downgraded till v0.6.0. looks like a basic problem

**Comment by phuslu** [comment]
should be fine in 0.9.5, please try https://go.dev/play/p/UDJOz1Neh92

let me look why is broken in fastdig cli

**Comment by phuslu** [comment]
fixed by using DoH in v0.9.6
So the root cause maybe related with UDP and possible no need to fix?
